### ISSUE TITLE
Not able to activate example function via checkoutServerExtensionCreate

### ISSUE THREAD

**Issue by h4gen** [Shopify/function-examples#164]
Hi,

I followed the example for [the cart and checkout functions ](https://shopify.dev/docs/apps/checkout/cart-checkout-validations). However in [Step 2](https://shopify.dev/docs/apps/checkout/cart-checkout-validations#step-2-activate-the-cart-and-checkout-validation-with-graphiql) when trying to activate it on the GraphQL API I get an error even though the function compiles correctly, is live and the correct Id is passed:

```
{
  "data": {
    "checkoutServerExtensionCreate": null
  },
  "errors": [
    {
      "message": "Access denied for checkoutServerExtensionCreate field. Required access: The shop must have access to checkout server extensions.",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "checkoutServerExtensionCreate"
      ],
      "extensions": {
        "code": "ACCESS_DENIED",
        "documentation": "https://shopify.dev/api/usage/access-scopes",
        "requiredAccess": "The shop must have access to checkout server extensions."
      }
    }
  ],
  "extensions": {
    "cost": {
      "requestedQueryCost": 10,
      "actualQueryCost": 10,
      "throttleStatus": {
        "maximumAvailable": 1000,
        "currentlyAvailable": 990,
        "restoreRate": 50
      }
    }
  }
}
```
Unfortunately [the API definition of the mutation](https://shopify.dev/docs/api/admin-graphql/unstable/mutations/checkoutServerExtensionCreate) gives also no more helpful information or clarifies what `access to checkout server extensions` exactly means.

[Someone else](https://community.shopify.com/c/shopify-apps/access-denied-for-checkoutserverextensioncreate-field-required/m-p/1951819#M59956) also posted the same problem in the community a few days ago.

 Interestingly the Query is totally indifferent to feeding rubbish as Id. Can somebody help or give a hint wat might go wrong?

Thanks,
Hagen

**Comment by h4gen** [comment]
Hi @jamesvidler and @nickwesselman,

Sorry to link you here, but we are really stuck. I can't find no help and I really think there is something wrong with the example. Can you please help? I linked you @nickwesselman as you did the last commit to this repo and @jamesvidler because you did the last commit to the cart-validation example function. I don't want to annoy, but would be really thankful for some help here. Especially, as I think this is valid feedback.

Best,
Hagen

**Comment by h4gen** [comment]
Hi @sjanczy2 👋 ,

thank you so much for getting back! I found the problem now. I was really missing the information that [developer preview](https://shopify.dev/docs/api/release-notes/developer-previews#previewing-new-features) is different from a normal dev store and needs a certain kind of store with the preview for that kind of extension enabled.

I have actually read this passage quit a few times but just from reading over it, it was not quite clear that point 5 actually does something different from a normal dev store. Only when following through I saw the difference in the UI.

Thank you so much!
